src/pages/home/release-notes/2022.md

```diff
@@ -262,12 +319,6 @@ For more details, see the documentation and release notes in the [aepsdk-react-n
 * Improved logging for troubleshooting.
 * Fixed an issue to ensure that event collection stops on session disconnection.
 
-## February 14, 2022
-
```

Comment:
This is also already in acp docs https://github.com/AdobeDocs/acp-mobile-sdkdocs/blob/main/src/pages/documentation/release-notes/2022.md#ios-identity-252